mobile/lib/modules/asset_viewer/views/gallery_viewer.dart

```diff
@@ -68,27 +76,51 @@ class GalleryViewerPage extends HookConsumerWidget {
       }
     }
 
-    void showInfo() {
-      showModalBottomSheet(
-        shape: RoundedRectangleBorder(
-          borderRadius: BorderRadius.circular(15.0),
-        ),
-        barrierColor: Colors.transparent,
-        backgroundColor: Colors.transparent,
-        isScrollControlled: true,
-        context: context,
-        builder: (context) {
-          return ExifBottomSheet(assetDetail: assetDetail!);
-        },
-      );
+    void precacheNextImage(int index) {
+      if (index < assetList.length && index > 0) {
+        final asset = assetList[index];
+        if (asset.isLocal) {
+          precacheImage(AssetEntityImageProvider(assetList[index].local!), context);
+        } else {
+          if (isLoadPreview.value) {
+            precacheImage(
+              CachedNetworkImageProvider(
+                getThumbnailUrl(asset.remote!),
```

Comment:
I believe we have to add an argument here to fetch the JPEG type (medium size) instead of the default WebP (very small size)

Reply:
So something like this 
![image](https://user-images.githubusercontent.com/27055614/215823978-7ffdaefd-3929-42fc-ae7b-aa727be8aa7a.png)

Notice that there is an appnended `api` because the `ThumbnailFormat` enum existing in both `photo_manager` library and the `open_api` library so you might have to import open api library like so `import 'package:openapi/api.dart' as api;`

Reply:
Thanks, this should be done now.